presto-main/src/main/java/com/facebook/presto/sql/planner/plan/StatisticAggregations.java

```diff
@@ -147,4 +172,32 @@ public StatisticAggregations getPartialAggregation()
             return partialAggregation;
         }
     }
+
+    @Immutable
+    public static class VariableAggregation
+    {
+        private final VariableReferenceExpression variableReferenceExpression;
```

Comment:
I wonder if it would be cleaner to maintain 2 parallel lists instead of a list of these structs.

List<VariableReferenceExpression> aggregateVariables;
List<Aggregation> aggregations;